src/RA_Defs.h

```diff
@@ -11,6 +11,8 @@
 #include <queue>
 #include <deque>
 #include <map>
+#include <memory>
+#include <atlbase.h>
```

Comment:
prefer to have this in the files that need it, rather than global. `RA_Defs` is included by everything, not just the UI.

Reply:
Sure, I was gonna try to a precompiled header (it will work, had it working in RASuite fork) for faster compile and preventing double includes.

I agree with you though.